src/Proto.Cluster/Identity/IdentityStorageLookup.cs

```diff
@@ -83,6 +85,10 @@ internal PID RemotePlacementActor(string address)
 
         public Task RemovePidAsync(PID pid, CancellationToken ct)
         {
+            if (_system.Token.IsCancellationRequested)
```

Comment:
silent exit if shutting down. this one was a major issue before.
RequestAsyncStrategy hitting this method when it detects that there is no activation